### PR TITLE
Feature/155/forgot passsword

### PR DESCRIPTION
**Pull request by tarasSanotskyi**
- [x] Create Forgot Password form
- [x] Create info popup
- [x] Add unit tests

image
image

### CODE REVIEW

File: tests/unit/containers/guest-home-page/forgot-password/ForgotPassword.spec.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    await waitFor(() => expect(img).toBeInTheDocument())
+  })
+
+  it('should submit form and open info popup', async () => {
+    const input = screen.getByLabelText(/common.labels.email/i)
+    fireEvent.change(input, { target: { value: 'test@gmail.com' } })
+
+    const button = screen.getByText('login.sendPassword')
+    fireEvent.click(button)
+    const resetpssTitle = screen.getByText('login.passwordReset')
```
Comment (rostyslaw27) [inline tests/unit/containers/guest-home-page/forgot-password/ForgotPassword.spec.js:49]: what is pss? maybe password

**Review by rostyslaw27 — CHANGES_REQUESTED** [review]

File: src/containers/guest-home-page/forgot-password/forgot-password.style.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  },
+  title: {
+    typography: 'h5',
+    fontWeight: 500,
+    mb: 1
+  },
+  description: {
+    typography: 'body2'
+  },
+  sentPass: {
```
Comment (rostyslaw27) [inline src/containers/guest-home-page/forgot-password/forgot-password.style.js:18]: sendPassword

**Review by rostyslaw27 — CHANGES_REQUESTED** [review]

File: tests/unit/containers/guest-home-page/forgot-password/ForgotPassword.spec.js (hunk 1 above)
Comment (tarasSanotskyi) [reply tests/unit/containers/guest-home-page/forgot-password/ForgotPassword.spec.js]: fixed

File: src/containers/guest-home-page/forgot-password/forgot-password.style.js (hunk 2 above)
Comment (tarasSanotskyi) [reply src/containers/guest-home-page/forgot-password/forgot-password.style.js]: fixed

**Review by rostyslaw27 — APPROVED** [review]

**Pull request merged**